harvest/algo.py

```diff
@@ -220,7 +220,7 @@ def default_param(self, symbol, interval, ref, prices):
             if interval == None:
                 interval = self.trader.interval
             if prices == None:
-                prices = self.trader.queue.get_symbol_interval_prices(symbol, interval, ref)
+                prices = self.trader.streamer.storage.load(symbol, interval)
```

Comment:
I should update this so that trader has an interface for the algo to fetch data instead of this chaining of variables, but for right now this replaced the queue.

Reply:
I just wrote this in the other thread too but IMO the data should be stored in `Trader`, since that was what the Trader class was made for more or less.

Reply:
Okay, I'll move the storage to the trader.

Reply:
Actually hold on a bit, I'm currently merging this PR with my rebase as well as making some adjustment I just said.

Reply:
I'll submit a PR soon and ask for a review